productivity-suite/app/fragments/todo-lists/src/dispatchSelectedListUpdated.ts

```diff
@@ -1,16 +1,7 @@
-import { dispatchPiercingEvent } from "piercing-library";
-import type { TodoList } from "shared";
+import { getBus } from "piercing-library";
 
-export function dispatchSelectedListUpdated(
-  el: Element,
-  listSelected: TodoList,
-  which?: "previous" | "next"
-) {
-  dispatchPiercingEvent(el, {
-    type: "todo-list-selected",
-    payload: {
-      list: listSelected,
-      which,
-    },
+export function dispatchSelectedListUpdated(el: Element, listName: string) {
```

Comment:
NIT, if you name the param `name` then you can write:

```ts
getBus(el).dispatch("todo-list-selected", {name});
```

Reply:
yeah...

the reason why I did it in this way is that in `dispatchSelectedListUpdated` I feel like `name` is slightly too generic but I don't want to use `listName` in the `todo-list-selected` payload as there on the other hand it would be too specific.

I think I kinda prefer it this way, but if you do quite prefer the `name` I'm ok with it 🙂

Reply:
PS: what do you think if I just move this function in the root of the todo-lists component (which is the only place that uses it)?

Reply:
done both 👍